draft-httpbis-doh-preference-hints.md

```diff
@@ -159,6 +159,21 @@ of the web server. Web servers MAY send multiple DoH-Preference headers. Web
 servers MUST NOT send the DoH-Preference header in HTTP responses conveyed over
 a non-secure transport.
 
+## Considerations For Choosing a Preferred DoH Server
+
+The choice of DoH server can affect overall performance and responsiveness as
+perceived by the client. Some example considerations in choosing a preferred
+DoH server are:
+
+* A DoH host specified as a host name rather than an IP address will require
+one or more additional DNS resolutions when the cached DNS entries for the
+resolver or resolvers expire.
+* Support for extension mechanisms (e.g. EDNS(0)) may be desired
+* Clients, particularly mobile device clients, may move frequently move between
+networks that have different network paths to the DoH server.
+* Specifying a DoH server for high traffic HTTP domains could result diverting a
+large number of DNS requests from the clients' standard DNS resolution mechanism
```

Comment:
The number of DNS requests is not really proportional to the amount of HTTP traffic when TTLs are high. And I'm not sure how one can act on this consideration?

Reply:
Indeed. for this and other reasons that occured to me this since, the thing I was thinking about didn't make sense. Removed this for now.